Installation/cmake/modules/CGALConfig_install.cmake.in

```diff
@@ -41,11 +44,19 @@ set(WITH_CGAL_ImageIO "@WITH_CGAL_ImageIO@" )
 set(WITH_CGAL_Qt3     "@WITH_CGAL_Qt3@" )
 set(WITH_CGAL_Qt4     "@WITH_CGAL_Qt4@" )
 
-set(CGAL_LIBRARY         "@CGAL_LIBRARY_INSTALLED@")
-set(CGAL_Core_LIBRARY    "@CGAL_Core_LIBRARY_INSTALLED@")
-set(CGAL_ImageIO_LIBRARY "@CGAL_ImageIO_LIBRARY_INSTALLED@")
-set(CGAL_Qt3_LIBRARY     "@CGAL_Qt3_LIBRARY_INSTALLED@")
-set(CGAL_Qt4_LIBRARY     "@CGAL_Qt4_LIBRARY_INSTALLED@")
+macro(CGAL_get_property_optional VAR TARGET PROPERTY)
+  if(TARGET ${TARGET})
+    get_property(${VAR} TARGET ${TARGET} PROPERTY ${PROPERTY})
+  else()
+    set(${VAR} "")
+  endif()
+endmacro()
+
+CGAL_get_property_optional(CGAL_LIBRARY TARGET CGAL::CGAL PROPERTY LOCATION)
+CGAL_get_property_optional(CGAL_Core_LIBRARY TARGET CGAL::CGAL_Core PROPERTY LOCATION)
+CGAL_get_property_optional(CGAL_ImageIO_LIBRARY TARGET CGAL::CGAL_ImageIO PROPERTY LOCATION)
+CGAL_get_property_optional(CGAL_Qt3_LIBRARY TARGET CGAL::CGAL_Qt3 PROPERTY LOCATION)
+CGAL_get_property_optional(CGAL_Qt4_LIBRARY TARGET CGAL::CGAL_Qt4 PROPERTY LOCATION)
```

Comment:
I am not sure, but it seems lines 55-59 will create a syntax error: the keywords `TARGET` and `PROPERTY` should probably be removed.